packages/driver/src/cy/commands/navigation.ts

```diff
@@ -894,6 +904,12 @@ export default (Commands, Cypress, cy, state, config) => {
 
           knownCommandCausedInstability = true
 
+          // if this is multi-domain, we need to tell the primary to change
+          // the AUT iframe since we don't have access to it
+          if (Cypress.state('isMultiDomain')) {
+            return Cypress.emit('multi:domain:visit:url', { url })
```

Comment:
Is there any sort of naming convention for these event names?

Reply:
Moved event to `specBridgeCommunicator` and renamed to `visit:url`. [5aafb8f](https://github.com/cypress-io/cypress/pull/20467/commits/5aafb8f3a723b6ddea1655fcc032e856ef1737d4)